app/models/product_user.rb

```diff
@@ -3,7 +3,7 @@ class ProductUser < ActiveRecord::Base
   belongs_to :user
   belongs_to :product
   belongs_to :product_access_group
-  belongs_to :approved_by_user, class_name: "User", foreign_key: "approved_by"
+  belongs_to :approved_by_user, class_name: "User", foreign_key: "approved_by", inverse_of: false
```

Comment:
What's this `inverse_of: false` for?

Reply:
It means there is no inverse_of. Mainly, it's a way to appease rubocop.